tests/menu/test_get.py

```diff
@@ -0,0 +1,27 @@
+import os
+import pytest
+import shutil
+from buildtest.defaults import TESTCONFIG_ROOT
+from buildtest.menu.get import clone
+from buildtest.utils.file import is_dir, create_dir
+
+
+def test_clone():
```

Comment:
This should use pytest tmp_path.

Reply:
Okay this is now fixed in https://github.com/HPC-buildtest/buildtest-framework/pull/270/commits/542e243641b7a88a091dac4fbaf5869fe99ccfa9